vultr/terraform-provider-vultr
# Add/improve import for kubernetes and node pool resources

## Description

* Adds node pool import state function
* Updates some state read on kubernetes resource to not trip re-creation when a cluster is imported
* Throw an error if a kubernetes cluster is imported and does not have the default node pool tag
* Document the imports for each 


### Checklist:


## Testing
* Create a VKE cluster outside of terraform
* Tag one of the node pools with `tf-vke-default`.  The terraform provider requires this tag for one (and only one) of the node pools in the cluster
* Create a terraform config for the cluster
```hcl
resource "vultr_kubernetes" "vke" {
    region  = "erw"
    label   = "gpu-test"
    version = "v1.30.0+1"

    node_pools {
        node_quantity = 3
        plan          = "vcg-a16-2c-8g-2vram"
        label         = "cust-np"
    }
} 
```
* Import the cluster
```
terraform import vultr_kubernetes.vke 519d7011-d02b-4264-828c-e30241c01408 
                                                                                                                            
vultr_kubernetes.vke: Importing from ID "519d7011-d02b-4264-828c-e30241c01408"...
vultr_kubernetes.vke: Import prepared!
  Prepared vultr_kubernetes for import
vultr_kubernetes.vke: Refreshing state... [id=519d7011-d02b-4264-828c-e30241c01408]

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```
* Verify the cluster is in your `terraform.tfstate` file
